src/cluster/src/install.rs

```diff
@@ -25,10 +27,10 @@ use crate::ClusterError;
 use crate::helm::{HelmClient, Chart, InstalledChart};
 use crate::check::{
     check_cluster_server_host, CheckError, check_helm_version, check_system_chart,
-    check_already_installed,
+    check_already_installed, _check_load_balancer_status,
 };
 
-const DEFAULT_NAMESPACE: &str = "default";
+pub const DEFAULT_NAMESPACE: &str = "default";
```

Comment:
Nit: Could this be `pub(crate)`?

Reply:
@vijaylaxmid can you make that change and also update branch?, we can merge after that